tox.ini

```diff
@@ -198,3 +199,16 @@ usedevelop = True
 commands =
     python setup.py sdist
     twine upload --skip-existing --non-interactive --config-file ./.pypirc dist/*
+
+[testenv:scripts]
+description = Integration tests of the builtin shell scripts
+deps =
+     {[testenv]deps}
+     pytest-testinfra
```

Comment:
Do we need to set a version constraint ? or is any version of testinfra working with this conftest ?

Reply:
I don't think that this is really necessary, testinfra appears to have a rather stable interface (plus tox will pull in the latest version anyway). I must confess that I have not tried other versions besides the latest though, so I can add a version constraint here if you'd like.

Reply:
That's fine. I haven't worked with testinfra and if we don't need a version constraint that's great. Just wanted to check with you. Thanks